modules/oe_content_project/config/install/core.entity_form_display.node.oe_project.default.yml

```diff
@@ -0,0 +1,299 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.oe_project.body
+    - field.field.node.oe_project.oe_author
+    - field.field.node.oe_project.oe_project_budget
+    - field.field.node.oe_project.oe_project_budget_eu
+    - field.field.node.oe_project.oe_project_calls
+    - field.field.node.oe_project.oe_project_dates
+    - field.field.node.oe_project.oe_project_departments
+    - field.field.node.oe_project.oe_project_result_files
+    - field.field.node.oe_project.oe_project_results
+    - field.field.node.oe_project.oe_project_website
+    - field.field.node.oe_project.oe_reference
+    - field.field.node.oe_project.oe_subject
+    - field.field.node.oe_project.oe_summary
+    - field.field.node.oe_project.oe_teaser
+    - node.type.oe_project
+  module:
+    - datetime_range
+    - entity_reference_revisions
+    - field_group
+    - link
+    - path
+    - rdf_skos
+    - text
+third_party_settings:
+  field_group:
+    group_result:
```

Comment:
Does a result file make sense without the result field filled in? I could not find anything related to this in the COMM specifications.

Reply:
I can't find anything of the sort in the requirements, we can leave it like this and, if necessary, we will add extra validation later on.